app-shells/fish/fish-3.5.1.recipe

```diff
@@ -56,12 +54,15 @@ defineDebugInfoPackage fish$secondaryArchSuffix \
 
 BUILD()
 {
-	export CFLAGS="-D_BSD_SOURCE"
-	export CXXFLAGS=$CFLAGS
-	export LIBS="-lnetwork -lexecinfo"
-
-	runConfigure ./configure \
-		--without-included-pcre2
+	mkdir -p build && cd build
+	cmake .. \
+		-DCMAKE_BUILD_TYPE=Release \
+		-DCMAKE_INSTALL_PREFIX=$prefix \
+		-DCMAKE_INSTALL_DATADIR=$dataDir \
+		-DCMAKE_INSTALL_SYSCONFDIR=$settingsDir \
+		-DCMAKE_INSTALL_DOCDIR=$docDir \
+		-DCMAKE_INSTALL_MANDIR=$manDir \
+		-DCMAKE_INSTALL_LOCALEDIR=$dataDir/locale
```

Comment:
Use $cmakeDirArgs instead of specifying these manually.

Reply:
Ah yes I'm a bit rusty, thanks